modules/backend/widgets/form/partials/_field.htm

```diff
@@ -5,9 +5,9 @@
         <?= $this->renderFieldElement($field) ?>
 
     <?php else: ?>
-        <?php
-            $fieldComment = $field->commentHtml ? trans($field->comment) : e(trans($field->comment));
-        ?>
+        <?php if ($field->comment): ?>
```

Comment:
This is merely where the value gets assigned, not where it actually gets used: https://github.com/octobercms/october/blob/master/modules/backend/widgets/form/partials/_field.htm#L18-L20

Reply:
Right, but it fails when running `trans($field->comment)` when `$field->comment = null`

I also considered reassigning the value in line:

```
<?php if (!$field->hidden): ?>

    <?php if (!$this->showFieldLabels($field)): ?>

        <?= $this->renderFieldElement($field) ?>

    <?php else: ?>
        <?php if ($field->label): ?>
            <label for="<?= $field->getId() ?>">
                <?= e(trans($field->label)) ?>
            </label>
        <?php endif ?>

        <?php if ($field->comment && $field->commentPosition == 'above'): ?>
            <p class="help-block before-field"><?= $field->commentHtml ? trans($field->comment) : e(trans($field->comment)) ?></p>
        <?php endif ?>

        <?= $this->renderFieldElement($field) ?>

        <?php if ($field->comment && $field->commentPosition == 'below'): ?>
            <p class="help-block"><?= $field->commentHtml ? trans($field->comment) : e(trans($field->comment)) ?></p>
        <?php endif ?>

    <?php endif ?>

<?php endif ?>
```